src/examples/mfem_common_plane.cpp

```diff
@@ -242,69 +240,62 @@ int main( int argc, char** argv )
   timer.stop();
   SLIC_INFO_ROOT(
       axom::fmt::format( "Time to set up elasticity bilinear form: {0:f}ms", timer.elapsedTimeInMilliSec() ) );
 
-  // This block of code sets up a central difference time integration scheme.
-  // The constructor takes a list of degrees of freedom in the velocity grid
-  // function that will have homogeneous boundary conditions applied to them.
-  mfem_ext::CentralDiffSolver solver{ ess_vdof_list };
+  // This block of code sets up a central difference time integration scheme. The constructor takes a list of degrees of
+  // freedom in the velocity grid function that will have homogeneous boundary conditions applied to them.
+  mfem_ext::CentralDiffSolver solver( ess_vdof_list );
   solver.Init( op );
 
   // This block of code does initial setup of Tribol.
   timer.start();
-  // Next, we create a Tribol coupling scheme between the contact surfaces on
-  // the MFEM mesh. To create the coupling scheme requires several steps: 1)
-  // building a boundary submesh, 2) building a LOR mesh (if required), 3)
-  // re-decomposing the domain to move spatially close surface element pairs on
-  // to the same rank, and 4) creating a Tribol mesh. Steps 1 and 2 are
-  // performed when this method is called. Steps 3 and 4 are accomplished via a
-  // call to updateMfemParallelDecomposition(), which is typically called before
-  // calling update().
+  // First, we create a Tribol coupling scheme between the contact surfaces on the MFEM mesh. To create the coupling
+  // scheme requires several steps: 1) building a boundary submesh, 2) building a LOR mesh (if required), 3)
+  // re-decomposing the domain to move spatially close surface element pairs on to the same rank, and 4) creating a
+  // Tribol mesh. Steps 1 and 2 are performed when this method is called. Steps 3 and 4 are accomplished via a call to
+  // updateMfemParallelDecomposition(), which is typically called before calling update().
   int coupling_scheme_id = 0;
   int mesh1_id = 0;
   int mesh2_id = 1;
-  tribol::registerMfemCouplingScheme( coupling_scheme_id, mesh1_id, mesh2_id, *pmesh, coords, contact_surf_1,
+  tribol::registerMfemCouplingScheme( coupling_scheme_id, mesh1_id, mesh2_id, mesh, coords, contact_surf_1,
                                       contact_surf_2, tribol::SURFACE_TO_SURFACE, tribol::NO_CASE, tribol::COMMON_PLANE,
-                                      tribol::FRICTIONLESS, tribol::PENALTY, tribol::BINNING_GRID );
-  // This API call adds a velocity field to the coupling scheme. This is used
-  // for computing the maximum common plane timestep and, if activated, gap rate
-  // penalty.
+                                      tribol::FRICTIONLESS, tribol::PENALTY, tribol::BINNING_BVH, exec_mode );
+  tribol::setMPIComm( coupling_scheme_id, MPI_COMM_WORLD );
+  // This API call adds a velocity field to the coupling scheme. This is used for computing the maximum common plane
```

Comment:
Should be "minimum" common plane time step. If the common plane timestep is less than the current explicit operators timestep then we ideally lower the dt and repeat the step.